test/integration/grpcroute_test.go

```diff
@@ -0,0 +1,32 @@
+package integration
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+func grpcEchoResponds(ctx context.Context, url string, hostname string) (bool, error) {
+	args := []string{
+		"-d",
+		`{"greeting": "Kong"}`,
+		"-insecure",
+		"-servername",
+		hostname,
+		url,
+		"hello.HelloService.SayHello",
+	}
+	stdout, stderr := new(bytes.Buffer), new(bytes.Buffer)
+
+	cmd := exec.CommandContext(ctx, "grpcurl", args...)
+	cmd.Stdout = stdout
+	cmd.Stderr = stderr
+
+	if err := cmd.Run(); err != nil {
+		return false, fmt.Errorf("failed to echo GRPC server STDOUT=(%s) STDERR=(%s): %w", strings.TrimSpace(stdout.String()), strings.TrimSpace(stderr.String()), err)
+	}
+
+	return stdout.String() == "{\n  \"reply\": \"hello Kong\"\n}\n", nil
```

Comment:
Might consider using a param for this instead of hardcoding.

Reply:
👍  see 5b957ecd3fc7e8311bfe8d3ac4eb9925e83597a0